bootstrap.php

```diff
@@ -60,6 +60,15 @@
     exit("Class Gdn not found.");
 }
 
+// Set up the dependency injection container.
+$dic = new \Garden\Container\Container();
+Gdn::setContainer($dic);
+
+$dic->setInstance('Garden\Container\Container', $dic)
+    ->rule('Interop\Container\ContainerInterface')
+    ->setAliasOf('Garden\Container\Container')
+;
```

Comment:
The config is a big chain and putting the semi-colon on a different line will help with merges.